bolt-sidecar/src/config/mod.rs

```diff
@@ -46,10 +46,18 @@ pub struct Opts {
     #[clap(long, env = "BOLT_SIDECAR_ENGINE_API_URL", default_value = "http://localhost:8551")]
     pub engine_api_url: Url,
     /// URL for the Constraint sidecar client to use
```

Comment:
Since we're touching the flags, what do you think of this description?

```suggestion
    /// URL to forward the constraints produced by the Bolt sidecar to a server supporting the Constraints API, such as an MEV-Boost fork.
```